plugin/arcconf/arcconf.py

```diff
@@ -543,44 +543,45 @@ def disks(self, search_key=None, search_value=None,
         return search_property(rc_lsm_disks, search_key, search_value)
 
     @_handle_errors
-    def volume_raid_create(self, name, raid_type, disks, strip_size,
-                           flags=Client.FLAG_RSVD):
+    def volume_raid_create(self, name, raid_type, disks, strip_size, flags=Client.FLAG_RSVD):
+        """
+
+        :param name: name of volume (this will be ignored)
+        :param raid_type: RAID Type of the LD
+        :param disks: Physical device
+        :param strip_size: Strip size
+        :param flags: for future use
+        :return: volume object of newly created volume
+
+        Depends on command:
+            arcconf create <ctrlNo> logicaldrive max <RAID LEVEL> <channel ID> <Device ID> <channel ID> <Device ID> \
```

Comment:
The text wideth should be less than 80.

Reply:
done - commit id - ab54bc5b54a37fedd78c6232d19408f1ea78c3a8